preload/resources/preload_helper.js

```diff
@@ -0,0 +1,16 @@
+function verifyPreloadAndRTSupport()
+{
+    var link = window.document.createElement("link");
+    var preloadSupported = link.relList && link.relList.supports("preload");
```

Comment:
hmm or maybe that results in a less clear message if `relList` is not supported. OK to leave as is